bundler/lib/dependabot/bundler/helpers.rb

```diff
@@ -20,21 +20,17 @@ def self.bundler_version(lockfile)
 
         if (matches = lockfile.content&.match(BUNDLER_MAJOR_VERSION_REGEX))
           matches[:version].to_i >= 2 ? V2 : V1
-        elsif Dependabot::Experiments.enabled?(:bundler_v1_unsupported_error)
-          DEFAULT
         else
-          failover_version
+          DEFAULT
         end
       end
 
       # If we are updating a project with a Gemfile.lock that does not specify
-      # the version it was bundled with, we failover to V1 on the assumption
-      # it was created with an old version that didn't add this information
+      # the version it was bundled with, we failover to V2
+      # the version it was bundled with, we failover to V2
       sig { returns(String) }
       def self.failover_version
-        return V2 if Dependabot::Experiments.enabled?(:bundler_v1_unsupported_error)
-
-        V1
+        V2
       end
```

Comment:
We are going to always return bundler v2 except if version defined lower than version `2`. When version is lower than version 2 we assume it is going to be V1 and our unsupported error is going to throw error for that.

Reply:
Is `failover_version` being used anywhere now? Maybe it can be removed.

Reply:
It is removed.